### PR TITLE
Number of turns left counter

### PR DESCRIPTION
**Pull request by kaushik-ck**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -300,17 +310,40 @@ def search_key():
                             if not haswon:
                                 #'you are out of turns!'
                                 done=True
-                                text=basicFont.render('GAME OVER', True, BLACK)
+                                textSurf9, textRect9 = text_objects("GAME OVER", numTurnsLeft)
```
Comment (vineetjc) [inline main.py]: This text doesn't fit the box

Comment (kaushik-ck) [reply main.py]: error

It fits in mine but I'll change the font size to 20. Would that be fine?

Comment (vineetjc) [reply main.py]: LGTM. I still have my doubts on whether this is the best way to do it, but I'm merging this PR. Good work.

**Comment by vineetjc** [comment]
@kaushik-ck Needs to resolve conflicts before I will merge.

**Comment by vineetjc** [comment]
Merging PR. @kaushik-ck good work

**Pull request merged**